packages/ag-charts-community/src/chart/interaction/gestureDetector.ts

```diff
@@ -61,7 +63,7 @@ export class GestureDetector extends BaseManager<GestureEventTypes, GestureEvent
     public constructor(private readonly domManager: DOMManager) {
         super();
         this.domManager.addEventListener('touchstart', this.touchstart, { passive: true });
-        this.domManager.addEventListener('touchmove', this.touchmove, { passive: true });
+        this.domManager.addEventListener('touchmove', this.touchmove);
```

Comment:
Can you confirm that the removal of this ` { passive: true } ` option does not regress [AG-10605](https://ag-grid.atlassian.net/browse/AG-10605)?

Reply:
Good shout - I've added `{ passive: false }` to suppress the warning & reflect that we might interrupt the browser default handling.